cinema/models.py

```diff
@@ -134,9 +130,7 @@ def save(
         )
```

Comment:
The `save` method is usually not the best place to put validation logic. The `save` method should only be responsible for saving the object. It's better to call the `full_clean` method before saving the object outside of the `save` method. This way, you ensure that the validation always happens before saving the object.